src/index.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import './index.scss';
+import { Provider } from 'react-redux';
+import { createStore, combineReducers } from 'redux';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import reducer from './store/reducer';
+import newsReducer from './store/newsReducer';
+import blogReducer from './store/blogReducer';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const rootReducer = combineReducers({
```

Comment:
please move the combineReducers and createStore process to a separate store initialization file in redux folder. it's for the separation of concerns between data layer and view layer, just in case maybe later on you decide to use Vue instead of React.